remoting/subsystem/src/main/java/org/jboss/as/remoting/RemotingTransformers.java

```diff
@@ -90,19 +93,56 @@ private void buildTransformers_2_1(ResourceTransformationDescriptionBuilder buil
                 .addRejectCheck(RejectAttributeChecker.DEFINED, ConnectorCommon.SERVER_NAME);
     }
 
+    //EAP 7.0
     private void buildTransformers_3_0(ResourceTransformationDescriptionBuilder builder) {
+        /*
+        ====== Resource root address: ["subsystem" => "remoting"] - Current version: 4.0.0; legacy version: 3.0.0 =======
+        --- Problems for relative address to root ["configuration" => "endpoint"]:
+        Different 'default' for attribute 'sasl-protocol'. Current: "remote"; legacy: "remoting"
+        --- Problems for relative address to root ["connector" => "*"]:
+        Missing attributes in current: []; missing in legacy [sasl-authentication-factory, ssl-context]
+        Missing parameters for operation 'add' in current: []; missing in legacy [sasl-authentication-factory, ssl-context]
+        --- Problems for relative address to root ["http-connector" => "*"]:
+        Missing attributes in current: []; missing in legacy [sasl-authentication-factory]
+        Missing parameters for operation 'add' in current: []; missing in legacy [sasl-authentication-factory]
+        --- Problems for relative address to root ["remote-outbound-connection" => "*"]:
+        Missing attributes in current: []; missing in legacy [authentication-context]
+        Different 'alternatives' for attribute 'protocol'. Current: ["authentication-context"]; legacy: undefined
+        Different 'alternatives' for attribute 'security-realm'. Current: ["authentication-context"]; legacy: undefined
+        Different 'alternatives' for attribute 'username'. Current: ["authentication-context"]; legacy: undefined
+        Missing parameters for operation 'add' in current: []; missing in legacy [authentication-context]
+        Different 'alternatives' for parameter 'protocol' of operation 'add'. Current: ["authentication-context"]; legacy: undefined
+        Different 'alternatives' for parameter 'security-realm' of operation 'add'. Current: ["authentication-context"]; legacy: undefined
+        Different 'alternatives' for parameter 'username' of operation 'add'. Current: ["authentication-context"]; legacy: undefined
+         */
+
+        builder.addChildResource(RemotingEndpointResource.ENDPOINT_PATH)
+                .getAttributeBuilder().setValueConverter(new AttributeConverter.DefaultAttributeConverter() {
+            @Override
+            protected void convertAttribute(PathAddress address, String attributeName, ModelNode attributeValue, TransformationContext context) {
+                if ("remote".equals(attributeValue.asString())) {
```

Comment:
so given that 6.2 also supports "remote" we don't need to transform this attribute at all.
I will update transformer to do so.